terraform/deployments/github/repos.yml

```diff
@@ -630,6 +630,13 @@ repos:
 
   search-v2-evaluator:
 
+  select-with-search-component:
```

Comment:
Terraform will try to create this repo but raise errors because it already exists. 

We need to import it into terraform state. Example code change: https://github.com/alphagov/govuk-infrastructure/commit/e5056f2193fd3e03954ee3d7bb325fc18f30c79f
You can update the current block (change repo name), as it's only used once.